### PR TITLE
feat: Stonks

### PR DESCRIPTION
**Pull request by BATMAH69**

### CODE REVIEW

File: ethereum-governance/src/subagents/treasury-swap/constants.ts [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+  },
+  {
+    address: stonksTestflightAddress,
+    from: "stETH",
+    to: "DAI",
+  },
+];
+export const ORDER_ADDRESS = orderAddress;
+
+export const STONKS_ORDER_CREATION: Record<string, any>[] = [];
```
Comment (katamarinaki) [inline ethereum-governance/src/subagents/treasury-swap/constants.ts:73]: Invalid type: value is an array, but type is specified as Record.

Comment (BATMAH69) [reply ethereum-governance/src/subagents/treasury-swap/constants.ts:73]: value array, specified array of record

File: ethereum-governance/src/common/constants.ts [hunk 2]
```diff
@@ -94,6 +94,29 @@ export const EMERGENCY_BRAKES_MS_ADDRESS =
...
+  "0x8Ba6D367D15Ebc52f3eBBdb4a8710948C0918d42";
+export const STONKS_USDC_DAI_ADDRESS =
+  "0x2B5a3944A654439379B206DE999639508bA2e850";
+export const STONKS_USDC_USDT_ADDRESS =
+  "0x278f7B6CBB3Cc37374e6a40bDFEBfff08f65A5C7";
+export const STONKS_USDT_DAI_ADDRESS =
+  "0x64B6aF9A108dCdF470E48e4c0147127F26221A7C";
+export const STONKS_USDT_USDC_ADDRESS =
+  "0x281e6BB6F26A94250aCEb24396a8E4190726C97e";
+export const STONKS_TESTFLIGHT_ADDRESS =
```
[the remark this answers is not in the document]
Comment (BATMAH69) [reply ethereum-governance/src/common/constants.ts:117]: It is our contract and it allowed to write e2e test
@rkolpakov what is your opinion?

File: ethereum-governance/src/subagents/acl-changes/constants.ts [hunk 3]
```diff
@@ -392,6 +392,13 @@ export const OWNABLE_CONTRACTS = new Map<string, IOwnable>([
       ownershipMethod: "proxy__getAdmin",
     },
   ],
+  [
+    ebOracleAddress,
```
[the remark this answers is not in the document]
Comment (BATMAH69) [reply ethereum-governance/src/subagents/acl-changes/constants.ts:396]: this was duplication

File: ethereum-governance/src/common/constants.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (rkolpakov) [reply ethereum-governance/src/common/constants.ts:115]: I'd prefer to remove it

[the remark this answers is not in the document]
Comment (BATMAH69) [reply ethereum-governance/src/common/constants.ts:115]: @katamarinaki @BATMAH69 removed testflight stonks, added skip for tests

**Review by katamarinaki — APPROVED** [review]
LGTM

**Pull request merged**